{"summary": {"n_tests": 7, "Pass": 2, "Failed": 4, "Warning": 0, "Error": 1}, "tests": [{"name": "Arrays", "doc": "Checks differences between arrays", "module": "Example", "result": "Failed", "diagnosis": {"type": "AssertionError", "description": "the arrays should be different but aren't", "place": "assert a == b, \"the arrays should be different but aren't\"", "output": ""}}, {"name": "Something else", "doc": "This is another that fails", "module": "Example", "result": "Failed", "diagnosis": {"type": "AssertionError", "description": "this other test is a fail case", "place": "assert a == b, \"this other test is a fail case\"", "output": ""}}, {"name": "Standard output", "doc": "This function should always fail and has standard output", "module": "Example", "result": "Failed", "diagnosis": {"type": "AssertionError", "description": "", "place": "assert 1 == 2", "output": "this is a standard output\n"}}, {"name": "This other thing", "doc": "This one also fails", "module": "Example", "result": "Failed", "diagnosis": {"type": "AssertionError", "description": "this test should be failing", "place": "assert a == b, \"this test should be failing\"", "output": ""}}, {"name": "Username", "doc": "", "module": "Example", "result": "Error", "diagnosis": {"type": "TypeError", "description": "test_username() missing 1 required positional argument: 'username'", "place": "routine()", "output": ""}}]}